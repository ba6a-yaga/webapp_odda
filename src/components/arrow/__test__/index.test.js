import React from 'react';
import ReactDOM from 'react-dom';
import renderer from "react-test-renderer"
import Arrow from '../index';


it("render correctly ", () => {
    const TextInputComponent = renderer.create(<Arrow></Arrow>).toJSON();
    expect(TextInputComponent).toMatchSnapshot();
});

it("render correctly with inverted icon", () => {
    const TextInputComponent = renderer.create(<Arrow iconInvert></Arrow>).toJSON();
    expect(TextInputComponent).toMatchSnapshot();
});
