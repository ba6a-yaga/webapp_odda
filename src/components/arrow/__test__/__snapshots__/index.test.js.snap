// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly  1`] = `
<span
  className="by-arrow icon-arrow-left "
/>
`;

exports[`render correctly with inverted icon 1`] = `
<span
  className="by-arrow icon-arrow-left by-icon-arrow-left"
/>
`;
